pyasic/data/pools.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class PoolMetrics:
+    """A dataclass to standardize pool metrics returned from miners.
+    Attributes:
+
+    accepted: Number of accepted shares.
+    rejected: Number of rejected shares.
+    get_failures: Number of failures in obtaining work from the pool.
+    remote_failures: Number of failures communicating with the pool server.
+    active: Indicates if the miner is connected to the stratum server.
+    pool_rejected_percent: Percentage of rejected shares by the pool.
+    pool_stale_percent: Percentage of stale shares by the pool.
+    """
+
+    accepted: int = field(init=False)
+    rejected: int = field(init=False)
+    get_failures: int = field(init=False)
+    remote_failures: int = field(init=False)
+    stratum_active: bool = field(init=False)
+
```

Comment:
May want to add `alive` here too, a pool can be either `alive` and `active` (currently in use), `alive` and `!active`, (available as a backup), or `!alive` and `!active` (connection failure, may need to be updated).

Reply:
Do you mean this?

alive : bool = None
Not_active : bool = None

Reply:
Just needs `alive` and `active`, since `not_active = True` is the same thing as `active = False`.

Reply:
Noted.